### PR TITLE
Add Mobile or Tablet client application types e2e tests

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/Connectivity.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                  Parameters)
+        {
+        }
+
+        [Fact]
+        public async Task Connectivity_SavePage()
+        {
+            var connectionSpeed = CommonActions.SelectRandomDropDownItem(Objects.Admin.AddSolutionObjects.ConnectivityDropdown);
+
+            int numCheckboxes = CommonActions.GetNumberOfCheckBoxesDisplayed();
```
Comment (bjss-chris) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/Connectivity.cs:38]: `numCheckBoxes`

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MemoryAndStorage.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```
Comment (bjss-chris) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MemoryAndStorage.cs:4]: Needed?

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MobileOrTabletDashboard.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        [Theory]
+        [InlineData("Supported operating systems")]
+        [InlineData("Connectivity")]
+        [InlineData("Memory and storage")]
+        [InlineData("Third-party components and device capabilities")]
+        [InlineData("Hardware requirements")]
+        [InlineData("Additional information")]
+        public void MobileOrTabletDashboard_RowDisplayed(string rowTitle)
+        {
+            CommonActions.GetTableRowCells(0).Should().ContainEquivalentOf(rowTitle);
```
Comment (bjss-chris) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MobileOrTabletDashboard.cs:39]: `0` can be omitted from `GetTableRowCells` as it's the default.

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs [hunk 4]
```diff
@@ -38,5 +38,7 @@ internal static class AddSolutionObjects
         internal static By ConnectivityDropdown => By.Id("SelectedConnectionSpeed");
 
         internal static By ResolutionDropdown => By.Id("SelectedScreenResolution");
+
+        internal static By MinimumMemoryDropdown => By.Id("SelectedMemorySize");
```
Comment (bjss-chris) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Objects/Admin/AddSolutionObjects.cs:42]: `DropDown`

**Review by bjss-chris — CHANGES_REQUESTED** [review]
A few very minor things.

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MemoryAndStorage.cs (hunk 2 above)
Comment (PhysicsJon) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MemoryAndStorage.cs]: Removed

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MobileOrTabletDashboard.cs (hunk 3 above)
Comment (PhysicsJon) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/ClientApplicationTypes/MobileOrTablet/MobileOrTabletDashboard.cs]: Removed

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**